docs/en/sql-reference/functions/other-functions.md

```diff
@@ -1172,7 +1172,104 @@ Result:
 
 ## finalizeAggregation {#function-finalizeaggregation}
 
-Takes state of aggregate function. Returns result of aggregation (finalized state).
+Takes state of aggregate function. Returns result of aggregation (finalized state). Useful with [-State](../../sql-reference/aggregate-functions/combinators.md#agg-functions-combinator-state) combinator.
```

Comment:
> Useful with [-State] combinator 

Why is it so? -State combinator returns a state of aggregate function. I'd better delete the whole sentence about -State combinator and just add several words in parentheses. For example "Returns result of aggregation (or finalized state if you use it with -State combinator).